test/runtests.bat

```diff
@@ -4,10 +4,12 @@ REM create bin directory if it doesn't exist
 if not exist ..\bin mkdir ..\bin
 
 REM compile the code into the bin folder
-javac  ..\src\seedu\addressbook\Addressbook.java -d ..\bin
+javac  ..\src\seedu\addressbook\util\FilePathUtil.java -d ..\bin\util ..\src\seedu\addressbook\Addressbook.java -d ..\bin
```

Comment:
I added this line due to compile problem with the batch file after adding FilePathUtil.java.